src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/FileLengthCheck.java

```diff
@@ -69,7 +70,8 @@
     private int max = DEFAULT_MAX_LINES;
 
     @Override
-    protected void processFiltered(File file, List<String> lines) {
+    protected void processFiltered(File file, FileText fileText) {
+        final List<String> lines = fileText.getLines();
```

Comment:
please avoid usage of temporal object - `List<String> lines` , that is easy refactoring.